src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -23,6 +25,8 @@ const (
 	kb = 1024
 	mb = 1024 * kb
 	gb = 1024 * mb
+
+	defaultSchemaRegistryPort = 8081
```

Comment:
I don't see this default documented anywhere, should we document it?

Reply:
Will do in the cluster_types.go